### PR TITLE
Use apptest lib

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#13206

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -217,25 +218,27 @@ github.com/garyburd/redigo v0.0.0-20150301180006-535138d7bcd7/go.mod h1:NR3MbYis
...
-github.com/giantswarm/apiextensions/v2 v2.0.0 h1:rKUHhiM8herDiNSgkfdLwqs+Pzrews2T/8JJ4Z4JCmA=
 github.com/giantswarm/apiextensions/v2 v2.0.0/go.mod h1:/qSx5jA2F5um0ipvVDMZJz+rJqsPZewzz4jjMXkXYFw=
-github.com/giantswarm/apiextensions/v2 v2.1.0 h1:2KoU6Kj4Oc1DcMu652bvaET1q5aGmh4D2aTptL9a2Ng=
-github.com/giantswarm/apiextensions/v2 v2.1.0/go.mod h1:/qSx5jA2F5um0ipvVDMZJz+rJqsPZewzz4jjMXkXYFw=
+github.com/giantswarm/apiextensions/v2 v2.4.0 h1:xHGA5cc0T2KEbOxHPAb8ZRbCUwpykHFYljKiyo+iSSw=
+github.com/giantswarm/apiextensions/v2 v2.4.0/go.mod h1:/qSx5jA2F5um0ipvVDMZJz+rJqsPZewzz4jjMXkXYFw=
 github.com/giantswarm/appcatalog v0.2.7 h1:SVD1OH/C5zEP8TaeS3L9t8+lVcfaRZhwY98sn8v9lYs=
 github.com/giantswarm/appcatalog v0.2.7/go.mod h1:WHO5lJvcuwzToxxHPdiX9CVxvMsmP8YZfjoo4E/NTLs=
+github.com/giantswarm/apptest v0.0.0-20200921104650-2e669229ae35 h1:STv3dsEVHwyoD5RDuY6JyEpjVVwFEE2LM0ME5qMCR94=
+github.com/giantswarm/apptest v0.0.0-20200921104650-2e669229ae35/go.mod h1:ziCoOxs9q70pwzpDXP9WpXnvg6JBg0WLTHi3s/MEJjk=
```
Comment (rossf7) [inline go.sum:227]: TODO: Needs https://github.com/giantswarm/apptest/pull/2 to be released.

File: cmd/bootstrap/runner.go [hunk 2]
```diff
@@ -27,11 +28,11 @@ import (
 )
 
 const (
-	appOperatorVersion            = "2.2.0"
+	appOperatorVersion            = "2.3.2"
 	chartMuseumName               = "chartmuseum"
 	chartMuseumCatalogStorageURL  = "http://chartmuseum-chartmuseum.giantswarm.svc.cluster.local:8080/charts/"
 	chartMuseumVersion            = "2.13.3"
-	chartOperatorVersion          = "2.3.1"
+	chartOperatorVersion          = "2.3.3"
```
Comment (rossf7) [inline cmd/bootstrap/runner.go:35]: Also bumping the operators to the latest releases.

File: cmd/bootstrap/runner.go [hunk 3]
```diff
@@ -272,36 +284,24 @@ func (r *runner) installAppCatalogs(ctx context.Context, k8sClients k8sclient.In
...
-				Namespace: namespace,
-				Version:   chartMuseumVersion,
+		apps := []apptest.App{
+			{
+				CatalogName:   helmStableCatalogName,
+				CatalogURL:    helmStableCatalogStorageURL,
+				Name:          chartMuseumName,
+				Namespace:     namespace,
+				Version:       chartMuseumVersion,
+				WaitForDeploy: false,
```
Comment (tomahawk28) [inline cmd/bootstrap/runner.go:300]:
```suggestion
```
It should be`false` as default, should we keep it?

**Review by tomahawk28 — APPROVED** [review]

File: cmd/bootstrap/runner.go (hunk 3 above)
Comment (rossf7) [reply cmd/bootstrap/runner.go:300]: Yes we can simplify. The flag is for using in integration tests when you want to wait for the app to be installed before starting the test.

**Pull request merged**